spec.html

```diff
@@ -289,7 +289,35 @@ <h1>Iterator.prototype</h1>
 
       <emu-clause id="sec-iteratorprototype.constructor">
         <h1>Iterator.prototype.constructor</h1>
-        <p>The initial value of %Iterator.prototype%.constructor is %Iterator%.</p>
+        <p>`Iterator.prototype.constructor` is an accessor property with attributes { [[Enumerable]]: *false*, [[Configurable]]: *true* }. The [[Get]] and [[Set]] attributes are defined as follows:</p>
+
+        <emu-clause id="sec-get-iteratorprototype-constructor">
+          <h1>get Iterator.prototype.constructor</h1>
+          <p>The value of the [[Get]] attribute is a built-in function that requires no arguments. It performs the following steps when called:</p>
+          <emu-alg>
+            1. Return %Iterator%.
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-set-iteratorprototype-constructor">
+          <h1>set Iterator.prototype.constructor</h1>
+          <p>The value of the [[Set]] attribute is a built-in function that takes an argument _v_. It performs the following steps when called:</p>
+          <emu-alg>
+            1. Let _O_ be ? RequireObjectCoercible(*this* value).
+            1. If _O_ is %Iterator.prototype%, then
+              1. Return *undefined*.
+            1. Let _desc_ be ? _O_.[[GetOwnProperty]](*"constructor"*).
+            1. If IsDataDescriptor(_desc_) is *true* and _desc_.[[Writable]] is *false*, then
+              1. Return *undefined*.
+            1. Let _newDesc_ be the PropertyDescriptor { [[Value]]: _v_, [[Writable]]: *true*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.
+            1. Perform ? _O_.[[DefineOwnProperty]](*"constructor"*, _newDesc_).
+            1. Return *undefined*.
```

Comment:
if we're really going to do this it should be an AO that many things can share, since the only things that potentially change are Iterator.prototype and the property name

Reply:
that can be an editorial thing we address in the PR to 262